backend/app.js

```diff
@@ -7,18 +7,18 @@ const articleRoutes = require('./routes/articles');
 const httpServer = createServer(app);
 require('dotenv/config');
 
-app.use(express.json());
-app.use('/api/users', userRoutes);
-app.use('/api/articles', articleRoutes);
-
 const {
     DB_USER,
     DB_PASSWORD,
     DB_NAME,
     PORT,
-    SALT_ROUND
+    BODY_MAX
```

Comment:
Why did `SALT_ROUND` has been removed?

Reply:
It's a useless line in this file, I didn't know how to clean it